stl/inc/xloctime

```diff
@@ -890,6 +917,17 @@ protected:
     }
 
 private:
+    static constexpr bool __CLRCALL_OR_CDECL _Is_valid_fmt_specifier(const char _Specifier) {
+        constexpr char _Valid_specifiers[] = "aAbBcCdDeFgGhHIjmMnprRStTuUVwWxXyYzZ";
+        for (const char _Valid_specifier : _Valid_specifiers) {
+            if (_Valid_specifier == _Specifier) {
+                return true;
+            }
+        }
+
+        return false;
+    }

```

Comment:
I was thinking about `std::binary_search` here, but I guess this would require movement to a .cpp file which as of now does not exist for xloctime. I'm accepting any guidelines or suggestions.